entries/bind.xml

```diff
@@ -34,7 +34,7 @@
   </signature>
   <desc>Attach a handler to an event for the elements.</desc>
   <longdesc>
-    <p>As of jQuery 1.7, the <a href="/on/"><code>.on()</code></a> method is the preferred method for attaching event handlers to a document. For earlier versions, the <code>.bind()</code> method is used for attaching an event handler directly to elements. Handlers are attached to the currently selected elements in the jQuery object, so those elements <em>must exist</em> at the point the call to <code>.bind()</code> occurs. For more flexible event binding, see the discussion of event delegation in <a href="/on/"><code>.on()</code></a> or <a href="/delegate/"><code>.delegate()</code></a>.</p>
+    <p>As of jQuery 3.0, <code>.bind()</code> has been deprecated. It was superseded by the <a href="/on/"><code>.on()</code></a> method for attaching event handlers to a document since jQuery 1.7, so its use was already discouraged. For earlier versions, the <code>.bind()</code> method is used for attaching an event handler directly to elements. Handlers are attached to the currently selected elements in the jQuery object, so those elements <em>must exist</em> at the point the call to <code>.bind()</code> occurs. For more flexible event binding, see the discussion of event delegation in <a href="/on/"><code>.on()</code></a> or <a href="/delegate/"><code>.delegate()</code></a>.</p>
```

Comment:
But why would anyone search for docs for this method? Most likely because they use `.delegate()` which can't be said about all the readers of the `.bind()` docs.

I think we shouldn't link to pages about deprecated APIs, they should serve first as a warning to not use the API and second as docs for the API for people that *already use* it in their code. If there is useful info about event delegation there that's not available on the `.on()` page it should just be moved to the `.on()` page so that it's not required to look at pages for deprecated APIs to learn something generally useful.

(to clarify, I wanted to have the `.delegate()` mention removed completely, not just the link to the page)